msteams-platform/messaging-extensions/build-bot-based-plugin.md

```diff
@@ -46,8 +46,14 @@ Before you get started, ensure that you're familiar with the following standards
 
 # [Visual Studio Code](#tab/visual-studio-code)
 
-> [!NOTE]
-> Teams Toolkit support for bot-based message extension is available only in Teams Toolkit pre-release version. Before you get started, ensure that you've installed a [Teams Toolkit pre-release version](../toolkit/install-Teams-Toolkit.md#install-a-pre-release-version).
+Before you get started, ensure that you install the following tools to build and deploy your message extension:
+
+* Install the installed the latest [Teams Toolkit pre-release version](../toolkit/install-Teams-Toolkit.md#install-a-pre-release-version).
+* Ensure that the Copilot feature flag is enabled. To enable the feature flag, follow these steps:
```

Comment:
Rather than "Copilot", how about using the whole name? "Develop Copilot Plugin"

Reply:
@yuqizhou77 , updated to  **Develop Copilot Plugin**.